charts/drupal/values.yaml

```diff
@@ -202,6 +202,9 @@ nginx:
   #   paths:
   #     - "^/sites/[^/]+/files/(?:css|js)/"
 
+  status:
```

Comment:
Let's change this key to `status_page` and add a brief comment above it to describe what this is.

Reply:
done. did the same for `php.fpm.status_page.enabled` to make it uniform.